src/arrays/referencing_array_and_its_elements/hard/q1/ALittleToTheRight.java

```diff
@@ -0,0 +1,28 @@
+package arrays.referencing_array_and_its_elements.hard.q1;
+
+/*
+Given the following array ( { 3, 4, 6, 21, 48, 42, 89, 2, 0, 91, 33 } ),
+write a program called ALittleToTheRight, with a method called LilToDaRight, that will shift the
+elements of the integer array forward by one. For example, the element at index 4 will move to index 5, and so on.
+Then, the last element of the array will be pushed to the front of the array. At the end, print the resulting array
+after shifting all the elements, with each value on a separate line.
+ */
+public class ALittleToTheRight {
+    public static void main(String[] args) {
+        int[] arr = {3, 4, 6, 21, 48, 42, 89, 2, 0, 91, 33};
```

Comment:
This question is actually a very good example of a java question where we need to change the question due to hard coding. For this one, you will need to add a scanner which accepts input from the user, and then you should be able to carry on with the question like normal.